src/transformer/utils/totara/program.php

```diff
@@ -0,0 +1,34 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace src\transformer\utils\totara;
+defined('MOODLE_INTERNAL') || die();
+
+function program(array $config, \stdClass $program, $lang) {
+    $programname = $program->fullname ? $program->fullname : 'A Totara program';
+
+    $object = [
+        'id' => $config['app_url'].'/totara/program/view.php?id='.$program->id,
+        'definition' => [
+            'type' => 'http://id.tincanapi.com/activitytype/lms/program',
```

Comment:
This activity type is not registered at https://registry.tincanapi.com/ and is not hosted on id.tincanapi.com. 

http://id.tincanapi.com/activitytype/learning-program is also not on registry, but I believe I have submitted it for consideration and I've definitely used it.

Reply:
In this plugin we have used `http://id.tincanapi.com/activitytype/lms/course` and `http://id.tincanapi.com/activitytype/lms/module`. Keeping with Programs being a function of the LMS I thought `http://id.tincanapi.com/activitytype/lms/program` would be appropriate.

Happy to change it to `http://id.tincanapi.com/activitytype/learning-program` though if that's what you recommend, I was just going for consistency.

Reply:
Consistency is important. 

@ryansmith94 @davidpesce any idea why http://id.tincanapi.com/activitytype/lms/course and http://id.tincanapi.com/activitytype/lms/module are used since these do not exist on http://id.tincanapi.com ?

Reply:
I can't recall why they were used, seems likely that they were based off something else that did/does exist on the registry.

Reply:
Is there any chance we can get this split off into a separate issue so the Totara event can be merged?

Reply:
Agreed that the course and module activity types should be discussed in a separate issue. We should probably decide what the activity type should be for the learning program though to avoid having to change that too.

Reply:
Since a "program" is quite a Totara specific activity, perhaps we should make a Totara specific activity type, what do you think @garemoko?

Reply:
I think it's a familiar concept in a few LMSs. I have used http://id.tincanapi.com/activitytype/learning-program before and I believe that's already in the queue for approval on registry.

Reply:
Okay cool